Source/Core/arrayFill.js

```diff
@@ -0,0 +1,64 @@
+/*global define*/
+define([
+        './DeveloperError',
+        './defaultValue',
+        './defined'
+    ], function(
+        DeveloperError,
+        defaultValue,
+        defined) {
+    'use strict';
+
+    /**
+     * Fill an array or a portion of an array with a given value.
+     * @private
+     *
+     * @param {Array} array The array to fill.
+     * @param {*} value The value to fill the array with.
```

Comment:
I think `{*}` is technically correct, but we use always use `{Object}` to indicate "anything" in Cesium doc.